src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 server.port=4056
 api.endpoint.extensions=/company/{companyNumber}/extensions/requests
-spring.data.mongodb.uri=mongodb://mongo-db1-statler.dev.aws.internal:27017/extension_requests
+spring.data.mongodb.uri=${SPRING_MONGO_DB}
```

Comment:
Why not use the global env values that already exist?
https://github.com/companieshouse/chs-configs/blob/develop/toro1/global_env#L28
so ${MONGO_DB_URL}/extension_requests

Reply:
First thing I looked for but could not find "mongo-db1-statler.dev.aws.internal" in global env

Reply:
Also worth noting that this value gets overwritten in the start script:
https://github.com/companieshouse/extensions-api/blob/master/start.sh#L36

Reply:
Ok updated.

Reply:
on further poking around.
There is this env variable declared in the app env already: EXTENSIONS_API_MONGODB_URL
just use that one

Reply:
Yes we can use this.